src/control/mod.rs

```diff
@@ -834,18 +843,17 @@ bitflags::bitflags! {
     }
 }
 
-#[repr(u32)]
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
 /// Target to alter the sequence of page flips
 ///
-/// These are the [`ffi::drm_sys::DRM_MODE_PAGE_FLIP_FLAGS`] bits
+/// These represent the [`ffi::drm_sys::DRM_MODE_PAGE_FLIP_TARGET`] bits
 /// of [`PageFlipFlags`] wrapped in a regular `enum` due to their
 /// mutual-exclusiveness.
 pub enum PageFlipTarget {
     /// Absolute Vblank Sequence
-    Absolute = ffi::drm_sys::DRM_MODE_PAGE_FLIP_TARGET_ABSOLUTE,
+    Absolute(u32),
     /// Relative Vblank Sequence (to the current, when calling)
-    Relative = ffi::drm_sys::DRM_MODE_PAGE_FLIP_TARGET_RELATIVE,
+    Relative(u32),
 }
```

Comment:
This type probably shouldn't be unified with `VblankWaitTarget`, if only for keeping the documentation neatly separated?

Fwiw I agree with you placing doc comments above `#[derive]` attributes, feel free to correct the same here.

Reply:
I did consider merging these, but I think it might be worth keeping them separate, even if only for clarity of documentation. I could go either way though.